packages/inventory/src/components/detail/AppInfo.js

```diff
@@ -1,16 +1,18 @@
+/* eslint-disable camelcase */
 import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components/components/esm/Skeleton';
 
 class AppInfo extends Component {
     render () {
-        const { activeApps, active, loaded } = this.props;
+        const { activeApps, active, loaded, componentsMapper, store } = this.props;
         const activeApp = activeApps.find(item => item.name === active.appName) || activeApps[0];
+        const Cmp = activeApp && activeApp.name ? componentsMapper?.[activeApp.name] || activeApp?.component : 'missing component';
         return (
             <Fragment>
                 { activeApp && <div className={ `ins-active-app-${activeApp.name}` }>
-                    { activeApp.component ? <activeApp.component /> : 'missing component' }
+                    { Cmp ? <Cmp store={store} /> : 'missing component'}
```

Comment:
I see a `missing component` node in the `Cmp` declaration. Feels like the condition is not necessary

Reply:
Good point. I've change the `Cmp` declaration.